forth_src/base.fs

```diff
@@ -106,8 +106,31 @@ then (to) ; immediate
 : 2drop ( a b -- )
 postpone drop postpone drop ; immediate
 
+top value oldtop
+start @ value oldstart
+
+: top! ( addr -- )
+latest @ swap top latest @ - 
+2dup - latest ! 
+over to top swap over - swap 1+ move ;
```

Comment:
Having new words added to base.fs looks like somewhat of a drawback to me. I would like to see the minimum RAM footprint of Durexforth smaller than it is today, so anything added (while the impact may not be great) is a tiny step in the wrong direction.

For the new words: SAVE-PRG, SAVE-PACK, RESTORE-FORTH, DSIZE, TOP and TOP! I think the goal should be to remove them, or move them to optional modules. If it's not possible or easily done, then I can live with that :-)

Reply:
SAVE-PRG can be moved, but SAVE-PACK is essential for both it and the build process.  It could be converted to a build-time-only module at a complexity cost.  As the deploy binary is packed, RESTORE-FORTH needs to remain in functionality if not name.  RESTORE-FORTH requires TOP and TOP!

I could see a RESTORE-FORTH that restores a marker before all of these words to purge them out at first boot.  I think I could make these (eventually) disappear, yeah.  I'll try that unless you prefer otherwise.

DSIZE is already gone, agreed that it was a waste.